webapp/src/websocket_handlers.ts

```diff
@@ -526,3 +535,37 @@ export function handleHostLowerHand(store: Store, ev: WebSocketMessage<{
         });
     }, HOST_CONTROL_NOTIFICATION_TIMEOUT);
 }
+
+export function handleHostRemoved(store: Store, ev: WebSocketMessage<{
+    call_id: string,
+    channel_id: string,
+    session_id: string,
+}>) {
+    const channelID = ev.data.channel_id;
+    const client = getCallsClient();
+    if (!client || client?.channelID !== channelID) {
+        return;
+    }
+
+    const sessionID = client.getSessionID();
+    if (ev.data.session_id !== sessionID) {
+        return;
+    }
+
+    handleCallEnd(store, {data: {channelID}} as unknown as WebSocketMessage<EmptyData>); // fake the wsMsg
```

Comment:
Would calling this result in the client thinking the call has actually ended? I don't think that should be the case as they should be able to see it ongoing from outside (profiles and all).

Reply:
What I really wanted was just the disconnect, so I'm keeping that. You're right that we don't want to lose the profiles. 
I noticed that when the session times out the client is technically disconnected, but it takes another 10 or so seconds for the user's own client to finally disconnect, and they are given a "call connection failed" error message. This is much cleaner.